pandas/tests/tseries/offsets/test_fiscal.py

```diff
@@ -158,17 +158,6 @@ def test_apply(self):
 
 class TestFY5253NearestEndMonth(Base):
 
-    def test_get_target_month_end(self):
-        assert (makeFY5253NearestEndMonth(
```

Comment:
are these tests wrong?

Reply:
No, but `get_target_month_end` is removed in this PR.